kyuubi-hive-jdbc-shaded/src/main/resources/META-INF/LICENSE

```diff
@@ -224,3 +224,9 @@ MIT License
 -----------
 org.slf4j:slf4j-api
 org.slf4j:jcl-over-slf4j
+
```

Comment:
we shaded arrow into `kyuubi-hive-jdbc-shaded.jar`, now the transitive deps of arrow changed, the `kyuubi-hive-jdbc-shaded/pom.xml`'s shading rules need to be updated